### ISSUE TITLE
Code Overflow and Page Breakout in Article Screen

### ISSUE THREAD

**Issue by bornmay** [forem/forem#9424]
### The Issue

While visiting the article page which has a code format in it.
example. ```   ```
```
Hii this is what causing the page breakout
```

And the scrolling isn't working in the code format, here is the Preview

![screencapture-dev-to-zinox9-git-github-cheatsheet-22ok-1595395137405](https://user-images.githubusercontent.com/62112099/88137641-9e402f80-cc09-11ea-9974-34decbcffc45.png)

Here is the page breakout 
![Screenshot from 2020-07-22 10-56-06](https://user-images.githubusercontent.com/62112099/88137869-1eff2b80-cc0a-11ea-9ea8-85a70f175eb5.png)

Tackled this issue in Each article page with the Code Syntax, from 16-18 hours.

Thank You

**Comment by rhymes** [comment]
Thanks for the issue! We'll take your request into consideration and follow up if we decide to tackle this issue.

To our amazing contributors: issues labeled `type: bug` are always up for grabs, but for feature requests, please wait until we add a `ready for dev` before starting to work on it.

To claim an issue to work on, please leave a comment. If you've claimed the issue and need help, please ping @forem/oss and we will follow up within 3 business days.

For full info on how to contribute, please check out our [contributors guide](https://github.com/forem/forem/blob/master/CONTRIBUTING.md).

**Comment by rhymes** [comment]
Hi @8bithemant, I believe this to be a duplicate of https://github.com/forem/forem/issues/9374

Let me know if this is not the case.

The scrolling issue seems to be different though, can you link the specific post?

**Comment by 8bithemant** [comment]
Hii, @rhymes, I checked for the issue #9374 and found it a bit similar, but the scrolling issue is not mentioned though.

Here is the link the post where I had scrolling kind of issue 

https://dev.to/zinox9/git-github-cheatsheet-22ok

**Comment by rhymes** [comment]
@8bithemant my scrolling works fine, it's just the text to be outside the container. What browser do you have? I use Firefox 79:

![scrolling_code](https://user-images.githubusercontent.com/146201/88152132-e0ff0900-cc03-11ea-88b4-44df58bb354a.gif)

**Comment by 8bithemant** [comment]
In my case I still have the scrolling issue Have a view
![Screenshot from 2020-07-22 20-51-03](https://user-images.githubusercontent.com/62112099/88195053-2c91d100-cc5d-11ea-923c-d32dfceb708f.png)


This is a fresh ss from my computer

**Comment by 8bithemant** [comment]
Hii, @rhymes also take a view of my Phone too..

![Screenshot_2020-07-23-07-34-20-13](https://user-images.githubusercontent.com/62112099/88246087-07ca4780-ccb7-11ea-92ad-4f118f83264d.jpg)
The screen breaking and kinda issue occurs here too...

**Comment by ludwiczakpawel** [comment]
@8bithemant can you give us a source (markdown?) for this part of your post? Maybe including that whole list under "Commiting"

**Comment by cmgorton** [comment]
This no longer appears to be an issue. I have tested on Chrome, Firefox, and Safari. I am going to close this issue but let me know if you continue to experience it. Thanks!
